src/generic/Matrix.jl

```diff
@@ -404,6 +404,18 @@ function Base.view(M::AbstractAlgebra.MatElem{T}, rows::Colon, cols::Colon) wher
    return view(M, 1:nrows(M), 1:ncols(M))
 end
 
+Base.firstindex(M::AbstractAlgebra.MatElem, i::Int) = 1
+
+function Base.lastindex(M::AbstractAlgebra.MatElem, i::Int)
+  if i == 1
```

Comment:
(Also, I don't know if we care about that, but AA tends to use 3 spaces for indentation).